package.json

```diff
@@ -92,6 +93,20 @@
           "Puppetfile"
         ],
         "configuration": "./languages/puppetfile-language-configuration.json"
+      },
+      {
+        "id": "yaml",
+        "aliases": [
+          "YAML",
+          "yaml"
+        ],
+        "extensions": [
```

Comment:
Given that these are only in `/tasks/*.yml` should this be a filename or path pattern?

Reply:
Removed it entirely, as there are other better yaml extensions